corehq/apps/dhis2/tasks.py

```diff
@@ -52,4 +53,4 @@ def send_datavalues(domain_name):
 )
 def send_datasets_for_all_domains():
     for domain_name in find_domains_with_toggle_enabled(toggles.DHIS2_INTEGRATION):
-        send_datavalues(domain_name)
+        send_datasets.apply(args=(domain_name,))
```

Comment:
Calling it normally makes sense. [The docs](http://docs.celeryproject.org/en/latest/reference/celery.app.task.html) gave me the impression that using `apply` was the right way to do this, but they don't explain the difference between using `apply` and calling it normally. If there's no difference, or if it's not important, calling it normally is simpler. I'll change it.